lib/base/functionwrapper.hpp

```diff
@@ -39,7 +39,7 @@ inline std::function<Value (const std::vector<Value>&)> WrapFunction(void (*func
 template<typename Return>
 std::function<Value (const std::vector<Value>&)> WrapFunction(Return (*function)(const std::vector<Value>&))
 {
-	return std::bind(function, _1);
+	return [function](const std::vector<Value>& wrappers) -> Return { return function(wrappers); };
```

Comment:
Are you sure about the name `wrappers`?

Reply:
And how about ``callbacks``.

Reply:
These aren’t necessarily wrappers or callbacks. Pick a generic name.